esrally/driver/runner.py

```diff
@@ -2068,6 +2071,62 @@ def __repr__(self, *args, **kwargs):
         return "delete-async-search"
 
 
+class OpenPointInTime(Runner):
+    async def __call__(self, es, params):
+        index = mandatory(params, "index", self)
+        response = es.open_point_in_time(index=index,
+                                         params=params)
+        id = response.get("id")
+        op_name = mandatory(params, "name", self)
+        # is name uniqueness enforced somewhere? this may collide with async searches
```

Comment:
Uniqueness is not enforced but we could add a check to `CompositeContext#put`.

Reply:
Ok, so have the context itself raise an error if the key already exists?  I think this is doubly supportive of the notion that close_pit should also pop the pit_id from the CompositeContext 👍

Reply:
Yes, sounds good to me.